src/tests/test_claim_detector.py

```
@@ -0,0 +1,121 @@
+@pytest.fixture
+def processed_text(text_processor):
+    """Fixture that provides the processed version of our test sentence."""
+    return text_processor.normalize_text(TEST_SENTENCE)
+
+class TestTextProcessor:
+    """Tests for the TextProcessor component."""
+    
+    def test_normalize_text(self, text_processor, processed_text):
+        """Test that text is properly normalized."""
+        assert processed_text == "The Earth is round."
+        
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)